osutil/fshelpers_test.go

```diff
@@ -0,0 +1,53 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2017 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package osutil
+
+import (
+	"io/ioutil"
+	"os"
+	"strconv"
+
+	. "gopkg.in/check.v1"
+)
+
+type groupFindGidOwningSuite struct{}
+
+var _ = Suite(&groupFindGidOwningSuite{})
+
+func (s *groupFindGidOwningSuite) TestSelfOwnedFile(c *C) {
+	self, err := RealUser()
+	c.Assert(err, IsNil)
+
+	f, err := ioutil.TempFile("", "testownedfile")
```

Comment:
You could write this slightly shorter with:
```
name := filepath.Join(c.MkDir(), "testownedfile")
err := ioutil.WriteFile(name, nil, 0644)
c.Assert(err, IsNil)
```

Reply:
Thanks, totally forgot about `c.MkDir()`